vocabs/oada/index.js

```diff
@@ -289,6 +289,21 @@ register('time', override('generic-data-value', {
   },
 }));
 
+register('led-status', override('generic-data-value', {
+  description: 'Status of LED light. Typically used for debugging applications ' +
+      'Could be used to represent on/off status or red/green status, etc',
+  type: Boolean,
+}));
+
+register('network-strength', override('generic-data-value', {
+  description: 'Strength of wireless network connection in -dBm ' +
+      'ranging from ~-10dBm to ~-100dBm',
+  type: Number,
```

Comment:
JSON schema does not allow you to say something is a number, and then say that it also has properties (i.e. is an object).  A generic data value is an object with a "value" key and usually a "units" key (the units typically live under the template in the data document).  i.e. this should not be a Number but rather an object with keys "value" and "units."  Since "generic-data-value" is already in that form, then you just need to override the units part of generic-data-value: i.e. just delete the "type: Number" line.  The value should be a string rather than a number, FYI, in order to prevent rounding issues between different systems (i.e. a number is not guaranteed reproduced the same on the destination as the source, but a string is guaranteed to be the same)